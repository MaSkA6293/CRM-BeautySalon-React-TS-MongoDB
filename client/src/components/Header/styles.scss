// $heightHeader: 70px;
// .header {
//   height: auto;
// }

// @media screen and (min-width: 425px) {
//   .header {
//     height: $heightHeader;
//     padding: 10px;
//     justify-content: center;
//     &__back {
//       height: $heightHeader;
//     }
//   }
// }
