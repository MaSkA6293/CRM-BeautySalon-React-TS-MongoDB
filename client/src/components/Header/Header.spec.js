import React from "react";
import Header from "./index";

const setUp = (props) => shallow(<Header {...props} />);

describe("all tests Home", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  describe("Home component", () => {
    it("should render Home component", () => {
      expect(component).toMatchSnapshot();
    });
  });
});
