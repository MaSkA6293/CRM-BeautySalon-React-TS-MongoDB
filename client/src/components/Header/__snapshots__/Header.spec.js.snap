// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all tests Home Home component should render Home component 1`] = `
<Fragment>
  <WithStyles(CssBaseline) />
  <WithStyles(ForwardRef(AppBar))
    className="header"
    position="fixed"
  >
    <WithStyles(ForwardRef(Toolbar)) />
  </WithStyles(ForwardRef(AppBar))>
  <WithStyles(ForwardRef(Toolbar))
    id="back-to-top-anchor"
  />
  <ScrollTop>
    <WithStyles(ForwardRef(Fab))
      aria-label="scroll back to top"
      color="secondary"
      size="small"
    >
      <KeyboardArrowUpIcon />
    </WithStyles(ForwardRef(Fab))>
  </ScrollTop>
</Fragment>
`;
